src/unit_tests/os_auth/test_auth_validate.c

```diff
@@ -214,7 +229,7 @@ static void test_w_auth_validate_data_register_limit(void **state) {
     for(unsigned i=0; i<10; i++) {
         snprintf(agent_name, 2048, "__agent_%d", i);
         response[0] = '\0';
-        err = w_auth_validate_data(response,ANY_IP, agent_name, NULL);
+        err = w_auth_validate_data(response,ANY_IP, agent_name, NULL, NULL);
```

Comment:
In fact, test_w_auth_validate_data_register_limit is no longer required.
It was implemented to test a deprecated keys limit. When the limit was eliminated, this test should have been eliminated too, but it was only modified and now it doesn´t verify anything.
This is a good moment to remove it.

Reply:
Great!
It was removed!